test/test_env.py

```diff
@@ -25,7 +25,7 @@ def check_py_pkgs(self, requirements_file):
             line = line.strip()
             if line == "" or line[0] == "#":
                 continue
-            pkg = re.split(r'[>=<]', line)[0]
+            pkg = re.split(r'[>=<\[]', line)[0]
```

Comment:
Would be nice to have a comment here to indicate that this regex effectively extracts the package name and excludes the version info.

Reply:
I've added a comment for this.